{
  "created_at": "2022-05-05T04:39:47.144Z",
  "title": "On April 7, 2022, a threat actor obtained access to a Heroku database",
  "url": "https://status.heroku.com/incidents/2413?updated",
  "author": "peterwallhead",
  "points": 418,
  "num_comments": 27,
  "_tags": [
    "story",
    "author_peterwallhead",
    "story_31269062",
    "front_page"
  ],
  "objectID": "31269062",
  "original_created_at": "2022-05-05T03:28:26.000Z",
  "image": "",
  "localize": [
    {
      "locale": "ja",
      "title": "2022年4月7日、脅威者はHerokuのデータベースへのアクセスを取得しました。"
    },
    {
      "locale": "zh",
      "title": "2022年4月7日，一个威胁行为者获得了对Heroku数据库的访问权。"
    },
    {
      "locale": "zh-Hant",
      "title": "2022年4月7日，一個威脅行爲者獲得了對Heroku數據庫的訪問權。"
    }
  ],
  "source_updated_at": 1651830309900
}